Scripts/Python/ki/xKIChat.py

```diff
@@ -272,7 +272,11 @@ def SendMessage(self, message):
                             cFlags.private = True
                             foundBuddy = True
                             # Remove the "/p buddyname" from the message.
-                            message = pWords[1][len(plyrName) + 1:]
+                            # due to casefold, len(plyrName) may not equal the length of what the user typed
+                            # count number of spaces in player name and split message based on that
+                            pNameWords = plyrName.split()
+                            pWords2 = pWords[1].split(" ",len(pNameWords))
+                            message = pWords2[len(pNameWords)]
```

Comment:
A simpler option here might be to use the original string instead of the casefolded one, e.g.:

```python
message = pWords[1][len(player.getPlayerName()) + 1:]
```